pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -651,6 +651,8 @@ func parseAppInstanceConfig(getconfigCtx *getconfigContext,
 		appinstancePrevConfigHash, configHash, Apps)
 	appinstancePrevConfigHash = configHash
 
+	devUUIDStr := (config.GetId().Uuid)
```

Comment:
Why the outer parenthesis? Looks strange even though it works from a syntax perspectibve.

Reply:
ok will remove those.